### PR TITLE
Provide PST updates, examples and documentation

### PR DESCRIPTION
**Pull request by berlincount**
... maybe deleting temp folders would also be nice, but hey ...

### CODE REVIEW

File: pst/example/example.proto [hunk 1]
```diff
@@ -0,0 +1,14 @@
+// Example protobuffer definition in the Public Domain.
```
Comment (vega113) [inline pst/example/example.proto]: Honestly - I am not sure it's a good idea to add an example source here. Would be great to add it to wiki or something like that.

File: pst/README.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+field definition via templates.
+
+## Building
+
+Generally, PST is an integral part of the Apache Wave build process, and not
+called directly. If you want to build it, you can run
+
+  `gradle build`
+
+but it'll usually will be built for your.
```
Comment (wisebaldone) [inline pst/README.md]: you*

**Comment by wisebaldone** [comment]
Do you mind closing this please.

File: pst/example/example.proto (hunk 1 above)
Comment (wisebaldone) [inline pst/example/example.proto]: I think this is fine, keeps it all in one place where if you have the source you have the docs.

**Comment by wisebaldone** [comment]
Sorry Ignore my last message, @berlincount were you able to fix the small grammer mistake please and If I get a +1 from @vega113  well move forward with this.

Thank you again for your contribution.

**Comment by vega113** [comment]
@wisebaldone Did you have a chance to check of this patch works? If so, I guess we merge it into master.

**Comment by wisebaldone** [comment]
Ill try and get around to it today.

Also gonna look how we can upgrade to proto3 and remove the pst tool completely.

**Comment by vega113** [comment]
Thanks
That would be cool.

**Comment by wisebaldone** [comment]
Looks good +1 though could not check with dist version since the master build the dist isnt functioning.

**Comment by vega113** [comment]
Can you merge it into master then?

**Pull request merged**